packages/styleUtils/typography/stories/text.stories.tsx

```diff
@@ -31,114 +98,36 @@ export const Default = () => (
-  const textAlignments = {
-    left: "left",
-    right: "right",
-    center: "center"
-  };
-  const align = select("align", textAlignments, "center");
-
-  return (
-    <Text align={align as React.CSSProperties["textAlign"]}>
-      Lorem Ipsum is simply dummy text of the printing and typesetting industry.
-      Lorem Ipsum has been the industry's standard dummy text ever since the
-      1500s, when an unknown printer took a galley of type and scrambled it to
-      make a type specimen book. It has survived not only five centuries, but
-      also the leap into electronic typesetting, remaining essentially
-      unchanged. It was popularised in the 1960s with the release of Letraset
-      sheets containing Lorem Ipsum passages, and more recently with desktop
-      publishing software like Aldus PageMaker including versions of Lorem
-      Ipsum.
-    </Text>
-  );
-};
-
-export const Wrap = () => {
-  const wrapVals = {
-    wrap: "wrap",
-    nowrap: "nowrap",
-    truncate: "truncate"
-  };
-  const wrap = select("wrap", wrapVals, "wrap");
+export const DefaultText = Template.bind({});
 
-  return (
-    <div>
-      <p>Use the Knobs panel to change how text wraps</p>
-      <div style={{ width: "300px" }}>
-        <Text wrap={wrap as WrapVals}>
-          Lorem Ipsum is simply dummy text of the printing and typesetting
-          industry. Lorem Ipsum has been the industry's standard dummy text ever
-          since the 1500s, when an unknown printer took a galley of type and
-          scrambled it to make a type specimen book. It has survived not only
-          five centuries, but also the leap into electronic typesetting,
-          remaining essentially unchanged. It was popularised in the 1960s with
-          the release of Letraset sheets containing Lorem Ipsum passages, and
-          more recently with desktop publishing software like Aldus PageMaker
-          including versions of Lorem Ipsum.
-        </Text>
-      </div>
-    </div>
-  );
+export const ResponsiveSize = Template.bind({});
+ResponsiveSize.args = {
+  size: {
+    default: "s",
+    medium: "m",
+    large: "l",
+    jumbo: "xl"
+  }
 };
 
-export const Size = () => {
-  const sizes = {
-    s: "s",
-    m: "m",
-    l: "l",
-    xl: "xl"
-  };
-  const size = select("size", sizes, "m");
-
-  return (
-    <Text size={size as FontSize}>
-      Use the Knobs panel to change the font size
-    </Text>
-  );
-};
-
-export const ResponsiveSize = () => {
-  return (
-    <Text
-      size={{
-        default: "s",
-        medium: "m",
-        large: "l",
-        jumbo: "xl"
-      }}
-    >
-      Change the width of your viewport to see the font size change
-    </Text>
-  );
-};
-
-export const Color = () => {
-  const colors = {
-    themeTextColorPrimary,
-    themeTextColorSecondary,
-    themeTextColorDisabled,
-    blue
-  };
-  const color = select("color", colors, themeTextColorPrimary);
+export const _SuccessText = args => (
```

Comment:
All of the different types of text have the same styles. Should they be different?

Reply:
They should be different, they're different components with their own appearance. Will fix!